Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -7,7 +7,7 @@
 	<key>pfm_app_url</key>
 	<string>https://1password.com</string>
 	<key>pfm_description</key>
-	<string>1Password 8 settings</string>
+	<string>1Password settings</string>
```

Comment:
Also need to bump the pfm_last_modified date, but thanks for fixing this.

Reply:
Fixed in [commit](https://github.com/ProfileCreator/ProfileManifests/pull/579/commits/b772ca633cf4bed43aa0f10a79ab457b21cd9ad4)